SharedPackages/DataBrokerProtectionCore/Tests/DataBrokerProtectionCoreTests/BrokerProfileJobActionTests.swift

```diff
@@ -458,6 +458,7 @@ final class BrokerProfileJobActionTests: XCTestCase {
 
     func testWhenExpectationActionFailsDuringScan_thenRetryOnce() async {
```

Comment:
This test case was supposed to test for this specific scenario, but it wasn’t wired up correctly and short-circuited some of the logic.